script/import_scripts/mbox.rb

```diff
@@ -234,7 +259,7 @@ def create_user_indices
   def clean_title(title)
     title ||= ""
     #Strip mailing list name from subject
-    title = title.gsub(/\[[^\]]+\]+/, '').strip
+    title = title.gsub(/\[#{LIST_NAME}\]/, '').strip
```

Comment:
I submitted a commit with that change. Do I need to make a new PR, or have you just not noticed y et?

Thanks.